Jenkinsfile

```diff
@@ -1,26 +1,5 @@
-pipeline {
-    options {
-        buildDiscarder(logRotator(numToKeepStr: '20'))
-        timeout(time: 1, unit: 'HOURS')
-    }
-    // cf. https://github.com/jenkins-infra/documentation/blob/master/ci.adoc
-    agent {
-        label 'docker'
-    }
-    tools {
-        jdk 'jdk8'
-        maven 'mvn'
-    }
-    stages {
-        stage('main') {
-            steps {
-                sh 'mvn -B clean verify -Dmaven.test.failure.ignore'
-            }
-            post {
-                success {
-                    junit '**/target/surefire-reports/TEST-*.xml'
-                }
-            }
-        }
-    }
-}
+buildPlugin(useContainerAgent: true, configurations: [
```

Comment:
No, this would defeat the whole purpose of the CI system, which is to actually run tests on a Docker-enabled agent!

Reply:
oops, TIL!